accname/name/comp_labelledby_hidden_nodes.html

```diff
@@ -0,0 +1,266 @@
+<!doctype html>
+<html>
+<head>
+  <title>Name Comp: Labelledby & Hidden Nodes</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<p>Tests hidden node name computation as part of the <a href="https://w3c.github.io/accname/#comp_labelledby">#comp_labelledby</a> portion of the AccName <em>Name Computation</em> algorithm.</p>
+
+<!--
+
+  These tests verify browser conformance with the following note as part of accName computation Step 2B:
+
+  "The result of LabelledBy Recursion in combination with Hidden Not Referenced means
+  that user agents MUST include all nodes in the subtree as part of
+  the accessible name or accessible description, when the node referenced
+  by aria-labelledby or aria-describedby is hidden."
+
+-->
+
+<h2>Testing with <code>display:none</code></h2>
+
+    <button aria-labelledby="a11" data-expectedlabel="foo bar" data-testname="button with aria-labelledby using display:none hidden span (with nested display:none span)" class="ex"></button>>x</button>
+    <span id="a11" style="display: none;">
+        foo
+        <span id="a12" style="display: none;">bar</span>
```

Comment:
Removing redundant display:none which is the same no matter how many times it's applied.
```suggestion
        <span id="a12">bar</span>
```